applications/rag/metadata.yaml

```diff
@@ -50,17 +38,17 @@ spec:
       - name: cluster_name
         varType: string
         required: true
+      - name: create_brand
+        description: Create Brand OAuth Screen
+        varType: bool
+        defaultValue: false
       - name: create_cluster
         varType: bool
-        required: true
+        defaultValue: false
```

Comment:
What does default = false do? Uncheck the create cluster box in the UI?

Reply:
false will showing exisiting cluster list first.. and then user can select new.. if create_cluster is required.

Reply:
Hm, but why? I think we want them to choose create cluster option if possible.